xyz.mx_moment.moment.yaml

```diff
@@ -0,0 +1,783 @@
+- name: python3-webencodings
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "webencodings==0.5.1"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/f4/24/2a3e3df732393fed8b3ebf2ec078f05546de641fe1b667ee316ec1dcf3b7/webencodings-0.5.1-py2.py3-none-any.whl
+    sha256: a0af1213f3c2226497a97e2b3aa01a7e4bee4f403f95be16fc9acd2947514a78
+- name: python3-yarl
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "yarl==1.4.2"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/89/e3/afebe61c546d18fb1709a61bee788254b40e736cff7271c7de5de2dc4128/idna-2.9-py2.py3-none-any.whl
+    sha256: a068a21ceac8a4d63dbfd964670474107f541babbd2250d61922f029858365fa
+  - type: file
+    url: https://files.pythonhosted.org/packages/7f/8f/b3c8c5b062309e854ce5b726fc101195fbaa881d306ffa5c2ba19efa3af2/multidict-4.5.2.tar.gz
+    sha256: 024b8129695a952ebd93373e45b5d341dbb87c17ce49637b34000093f243dd4f
+  - type: file
+    url: https://files.pythonhosted.org/packages/d6/67/6e2507586eb1cfa6d55540845b0cd05b4b77c414f6bca8b00b45483b976e/yarl-1.4.2.tar.gz
+    sha256: 58cd9c469eced558cd81aa3f484b2924e8897049e06889e8ff2510435b7ef74b
+- name: python3-zipp
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "zipp==3.2.0"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/c4/79/3b770d51254a31bb85ba56ea70d7428d0c2c659a233cc9722352e028b539/zipp-3.2.0-py3-none-any.whl
+    sha256: 43f4fa8d8bb313e65d8323a3952ef8756bf40f9a5c3ea7334be23ee4ec8278b6
+
+- name: python3-socks
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "python-socks==1.2.2"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/f7/39/ce05e2772d9ec266644484ced4f9f6207b488b7670adcc9ddacf8ca78f91/python_socks-1.2.2-py3-none-any.whl
+    sha256: 41100508d6a3723c2e9884d3330d8bd022a7ef0a953e826637833bf882550fc7
+
+- name: python3-aiohttp-socks
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "aiohttp-socks==0.6.0"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/9a/6c/d302e5a8097fee1e83b9f8e9da10d7752fbf27c74db18b3cc9528b3479be/aiohttp_socks-0.6.0-py3-none-any.whl
+    sha256: db7aa48c0758ee45d7dbc1fde499912ec6fb77eab77a6e2808825d1f41d4e300
+
+- "python3-matrix-nio.json"
+
+- name: moment
+  buildsystem: qmake
+  post-install:
+  # This works around a bug in appstream-glib where it won't find the icon if
+  # /usr exists. For some reason.
+  - ln -s /app/share /app/usr
```

Comment:
can't you instead patch the build system to no put stuff in `usr`?

Reply:
I'm not sure how to do it. I stole this workaround from Pure Maps: https://github.com/flathub/io.github.rinigus.PureMaps/blob/2d8797f2837774b11e7b3c2eab3946622ba8948e/io.github.rinigus.PureMaps.json#L160-L162

Reply:
Is there any chance we could let this be like this? It is a bit hacky, but I don't see any negative side-effects of it. I can try to look into it later, but we would like to formally announce Moment soon and we'd like to have a Flatpak easily available when we do. I tried looking into the qmake files for pyotherside (which is the guilty dependency) and I couldn't find any references to "usr".

Reply:
it's not a blocker.

Reply:
it's not a blocker

Reply:
Let's leave it as-is for now then.